service.py

```diff
@@ -687,6 +687,11 @@ def author(author_id):
     comments = Comment.query.order_by(Comment.time.desc()).filter_by(author_id=author_id).all()
     return render_template('author.html', articles=articles, comments=comments, Tool=Tool)
 
+@app.route('/recent')
+def example():
```

Comment:
@sanoussibarkay 

`example` is a bad function name.  Use something like `sort_articles` to reflect the purpose of this function.

It seems that you sorted the articles by their post time in this function, but did not
explicitly get recent articles.    Note that recent articles consist of a subset of the sorted articles.

**Hui**



**Hui**